app/views/users/mailer/notification.html.erb

```diff
@@ -18,8 +21,12 @@
     <% # We assume the system notification is clean %>
     <%= @notification.message.html_safe %>
   <% elsif @notification.deliver? %>
```

Comment:
I guess this conditional could be refactored also, and this logic could be done outside the view and stored into `@notification.message`? My point being, that in both conditional branches there should be the `notifications.deliver.email_end_greeting` locale.